diffusers/sd_textual_inversion_training.ipynb

```diff
@@ -59,8 +59,7 @@
       "source": [
         "#@title Install the required libs\n",
         "!pip install -U -qq git+https://github.com/huggingface/diffusers.git\n",
-        "!pip install -qq accelerate transformers ftfy\n",
-        "!pip install -qq \"ipywidgets>=7,<8\""
+        "!pip install -qq accelerate transformers ftfy",
```

Comment:
The extra comma here is causing an issue with opening the notebook (invalid JSON)

Reply:
Thanks @paras-malhotra! Fixed in https://github.com/huggingface/notebooks/pull/304. Unfortunately I just saw your comment now, could have saved me a few minutes :)